Add description for NuGet host verification library (#2981)
openenclave/openenclave: cmake/HostVerifyNuGetDescription.txt

# Copyright (c) Open Enclave SDK contributors.

> # Licensed under the MIT License.

Reviewer: If this file goes directly to Nuget description, the copyright statements can be omitted.

Open Enclave Host-Side Verification Library is an SDK for building applications

> in order to perform host-side enclave verification in C.

Reviewer: and C++

> When an application is doing host-side enclave verification, it means that a
> host application is trying to authenticate a remote enclave's hardware and
> software settings so that the application can determine whether or not to trust
> the remote enclave.

Reviewer 2: Suggest to simplify this. Also the enclave doesn't have to be remove. Could be on the same machine just a different process.

```suggestion
Host-side enclave verification means that a non-enclave application is trying to
authenticate an enclave's hardware and software settings so that the application
can determine whether or not to trust the enclave.
```
